pgx/src/trigger_support.rs

```diff
@@ -11,6 +11,21 @@ Use of this source code is governed by the MIT license that can be found in the
 
 use crate::{is_a, pg_sys};
 
+/// Indicates which trigger tuple to convert into a [crate::PgHeapTuple].
+#[derive(Debug, Copy, Clone)]
+pub enum TriggerTuple {
+    /// The row for which the trigger was fired. This is the row being inserted, updated, or deleted.
+    /// If this trigger was fired for an INSERT or DELETE then this is what you should return from the
```

Comment:
"then this is what you" -- So I return the enum from the function?

Reply:
These were copy-pasta'd from Postgres' docs.  To me it's clear that you'd return the underlying tuple provided to you when you got the one specified from the enum variant.